.codecov.yml

```diff
@@ -29,7 +29,8 @@ coverage:
                                  # options: success, error, failure
         if_not_found: success
         if_ci_failed: error
-
+  
   ignore:          # files and folders that will be removed during processing
-    - "AlicerceTests/*"
+    - AlicerceTests*
+    - *.xctest
```

Comment:
Do we still need this? It didn't have any effect on our tests since codecov looks at the entire `Coverage.profdata` file which in our case has coverage for both targets (Alicerce.framework and AlicerceTests.xctest)